webrtc.html

```diff
@@ -8857,6 +8859,58 @@ <h2 id="sec.stats-model">Statistics Model</h2>
       [[!WEBRTC-STATS]].</p>
     </section>
     <section>
+      <h3>The stats selection algorithm</h3>
+      <p>
+	The <dfn>stats selection algorithm</dfn> is as follows:
+	<ul>
+          <li>
+            Let <var>result</var> be an empty RTCStatsReport.
+          </li>
+	  <li>If the selector is a MediaStreamTrack, add the following
+            objects to <var>result</var>, if they exist:
+	    <ul>
+	      <li>
+		The RTCMediaStreamTrackStats corresponding to the selector.
+	      </li>
+	      <li>
+		If the selector track is inbound, the RTCInboundRTPStreamStats
+                corresponding to the selector that has "remote=false", and the
+                RTCOutboundRTPStreamStats corresponding to the selector that has
+                "remote=true".
+	      </li>
+	      <li>
+		If the selector track is outbound, the RTCOutboundRTPStreamStats
+                corresponding to the selector that has "remote=false", and the
+                RTCInboundRTPStreamStats corresponding to the selector that has
+                "remote=true".
+	      </li>
+              <li>
+                The stats objects for all MediaStreams that have the
+                MediaStreamTrack as a member.
```

Comment:
I would prefer if you refer to the stats object by type, i.e. "The RTCMediaStreamStats objects for all..." as opposed to "the stats objects", that is less ambiguous. nit: You could also say that have the RTCMediaStreamTrackStats.id as a member of trackIds to be clear what "as a member" means.

Note for reviewers: This will expose IDs of RTCMediaStreamTrackStats that we are not interested in, meaning broken references. I think we should include the stream the track belongs to. I don't think we should include other tracks.